spec/services/search/postgres/user_spec.rb

```diff
@@ -0,0 +1,117 @@
+require "rails_helper"
+
+# rubocop:disable Rails/PluckId
```

Comment:
Just out of curiosity, why did you disable the cop instead of using `.ids`?

```ruby
User.ids == User.pluck(:id)
#=> true
```

Both generate the exact same query, i.e. `SELECT "users"."id" FROM "users"`.

Reply:
Because unfortunately I'm using `.pluck(:id)` on arrays of hashes, and Rubocop isn't smart enough to notice the difference. I wish somebody would fix it upstream :D

Reply:
> Rubocop isn't smart enough to notice the difference

Neither was I, apparently. 😉